app/retail/templates/shared/top_nav.html

```diff
@@ -18,7 +18,7 @@
 {% load i18n static matches %}
 <div class="top-nav g-font-muli {{class}}">
   <a class="link d-md-block d-none" href="{% url 'products' %}">Grow Open Source</a>
-  <a class="{% if request.path == '/' %}selected{% endif %}" href="/">
+  <a class="{% if request.path == '/' %}selected{% endif %}" href="{% url 'funder' %}">
```

Comment:
Should we keep these consistent? 🤔

Reply:
good catch.  will update IFF @PixelantDesign is okay with the direction i'm taking this.